src/dbnode/network/server/tchannelthrift/errors/errors.go

```diff
@@ -59,6 +62,43 @@ func NewBadRequestError(err error) *rpc.Error {
 	return newError(rpc.ErrorType_BAD_REQUEST, err, int64(rpc.ErrorFlags_NONE))
 }
 
+var (
+	reRPCError = regexp.MustCompile(`Error\({Type:(\w+) Message:(.*) Flags:(\d)}\)`)
```

Comment:
Probably a bad idea to have a `MustCompile` in an init, maybe hide it behind a `sync.Once()` or something like that?

Reply:
Won't the service panic either way? Seems like it might be preferable to have fail fast and have it happen on package load vs method call